src/modules/ce/components/ReferralNav.tsx

```diff
@@ -0,0 +1,150 @@
+import { ListRounded, RadioButtonUncheckedRounded } from '@mui/icons-material';
+import Person from '@mui/icons-material/Person';
+import { Container, Divider, Stack, Typography } from '@mui/material';
+import React, { useMemo } from 'react';
+import { Outlet, useOutletContext } from 'react-router-dom';
+import ButtonLink from '@/components/elements/ButtonLink';
+import Loading from '@/components/elements/Loading';
+import { DetailsIcon } from '@/components/elements/SemanticIcons';
+import CommonAppBar from '@/components/layout/CommonAppBar';
+import {
+  CONTEXT_HEADER_HEIGHT,
+  REFERRAL_HEADER_HEIGHT,
+  STICKY_BAR_HEIGHT,
+} from '@/components/layout/layoutConstants';
+import NotFound from '@/components/pages/NotFound';
+import useCurrentPath from '@/hooks/useCurrentPath';
+import useSafeParams from '@/hooks/useSafeParams';
+import ReferralStatusChip from '@/modules/ce/components/ReferralStatusChip';
+import { clientBriefName } from '@/modules/hmis/hmisUtil';
+import { ProjectDashboardRoutes } from '@/routes/routes';
+import {
+  CeReferralFieldsFragment,
+  useGetCeReferralQuery,
+} from '@/types/gqlTypes';
+import { generateSafePath } from '@/utils/pathEncoding';
+
+interface Props {}
+
+const ReferralNav: React.FC<Props> = ({}) => {
+  const { projectId, opportunityId, referralId } = useSafeParams() as {
+    projectId: string;
+    opportunityId: string;
+    referralId: string;
+  };
+
+  const {
+    data: { ceReferral: referral } = {},
+    loading,
+    error,
+  } = useGetCeReferralQuery({
+    variables: {
+      id: referralId,
+    },
+  });
+
+  const outletContext: ReferralContext | undefined = useMemo(
+    () => (referral ? { referral } : undefined),
+    [referral]
+  );
+
+  const currentPath = useCurrentPath();
+
+  if (loading) return <Loading />;
+  if (error) throw error;
+  if (!referral) return <NotFound />;
+
+  return (
+    <>
+      <CommonAppBar height={REFERRAL_HEADER_HEIGHT}>
```

Comment:
Reading this, it feels a bit weird to me to have three "AppBar" components on the page. The MUI AppBar by default would use a `header` component, but ours is just a div, so I think its just a question of naming. We could call it `CommonStickyBar` or something instead?
And/or maybe add a comment to the component explaining that it's used for several purposes any time we need a sticky bar, including the nav bar containing breadcrumbs, and sticky headers inside dashboard content